apps/staking/lib/metadata.ts

```diff
@@ -1,10 +1,9 @@
-import type { WagmiMetadata } from '@session/wallet/lib/wagmi';
 import { BASE_URL } from '@/lib/constants';
 
 const SITE_TITLE = 'Session Staking Portal';
 const SITE_DESCRIPTION = 'Stake and get rewarded for securing the Session Network.';
 const SITE_IMAGE = `${BASE_URL}/images/link_preview.png`;
-const SITE_ICON = `${BASE_URL}/images/icon.png`;
+// const SITE_ICON = `${BASE_URL}/images/icon.png`;
```

Comment:
What happens with this?

Reply:
its commented out in case its needed again

Reply:
Fine. Resolving